frontend/modules/common/CustomSuperagent.js

```diff
@@ -0,0 +1,14 @@
+import AuthStore from '../account/stores/AuthStore'
+var defaults = require('superagent-defaults');
```

Comment:
Why not `import defaults from 'superagent-defaults'`? It looks kinda odd to mix up two different import styles.